tests/components/switchbot/test_config_flow.py

```diff
@@ -85,6 +91,62 @@ async def test_bluetooth_discovery_requires_password(hass):
     assert len(mock_setup_entry.mock_calls) == 1
 
 
+async def test_bluetooth_discovery_lock_key(hass):
+    """Test discovery via bluetooth with a lock."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN,
+        context={"source": SOURCE_BLUETOOTH},
+        data=WOLOCK_SERVICE_INFO,
+    )
+    assert result["type"] == FlowResultType.MENU
+    assert result["step_id"] == "lock_chose_method"
+
+    result = await hass.config_entries.flow.async_configure(
+        result["flow_id"], user_input={"next_step_id": "lock_key"}
+    )
+    await hass.async_block_till_done()
+    assert result["type"] == FlowResultType.FORM
+    assert result["step_id"] == "lock_key"
+    assert result["errors"] == {}
+
+    with patch("switchbot.SwitchbotLock.verify_encryption_key", return_value=False):
```

Comment:
We should patch where the function is imported

```suggestion
    with patch("homeassistant.components.config_flow.SwitchbotLock.verify_encryption_key", return_value=False):
```

Reply:
There are a few other places to adjust as well